demisto_sdk/commands/split/tests/ymlsplitter_test.py

```diff
@@ -276,6 +277,7 @@ def test_extract_code(tmpdir):
         assert "import demistomock as demisto  #" not in file_data
         assert "from CommonServerPython import *  #" not in file_data
         assert "register_module_line" not in file_data
+        assert "### pack version 1.0.3" not in file_data
```

Comment:
add an assertion that "pack version 1.0.3" is in the original file, to make sure this nobody deletes it by mistake